Index.md

```diff
@@ -68,3 +68,4 @@
 | [Breaking Bad Characters](https://github.com/khushi-purwar/Web-dev-mini-projects/tree/main/breaking-bad-characters)                 |  A single page web application that uses the breaking bad API to display and filter characters from the show | 
 | [Simon Game](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/Simon_Game)                            |A basic memory game in which there are 4 boxes of different colors and one box is blinked. The user has to click the boxes blinked and also has to click the previous blinked boxes as well.|
 | [Javascript Color Change App](https://github.com/Ayushparikh-code/Web-dev-mini-projects/tree/main/JAVASCRIPT%20BUTTON%20APP) | It is a basic web app where the user can select a color from the options that are given on the web app and as he clicks on that option, all the buttons background color changes to that color. |
+| [Weight Converter]()                                                                                                             |A web page where user can convert weights from kilograms to grams, pounds and ounces.
```

Comment:
Add the link please? @palaksinghania05

Reply:
Yeah. I left it empty because it was not merged in this repository so the link was giving 4040 error.

Reply:
Please add the link now so that I can resolve and merge your PR. Refer to other project details above to get an idea! Thanks.

Reply:
added!!